app/components/HighlightBox.tsx

```diff
@@ -0,0 +1,15 @@
+import Background from "./Background";
+import InfoBox, { InfoBoxProps } from "./InfoBox";
+
+export type HighlightBoxProps = InfoBoxProps;
+
+export default function HighlightBox(props: HighlightBoxProps) {
```

Comment:
Do we really need a single component here? We had the discussion in the past that we prefer to stick to basic building blocks to keep the markup readable and flexible.

Reply:
Alright. Would you add a tech debt ticket in our backlog to replace all the existing occurrences of that pattern so that we do not mix both approaches?